File: c++/8-struktury.cpp
#include <stdio.h>
struct samochod
{
	const char * marka;
	int rok_produkcji;
};

void pokaz(struct samochod *s)
{
	printf("marka: %s\n", s->marka);
	printf("rok produkcji: %d\n", s->rok_produkcji);
}

int main()
{
	struct samochod volvo;
	
	volvo.marka = "volvo v40";
	volvo.rok_produkcji = 2010;

	pokaz(&volvo);
	return 0;
}

File: go/8-struktury.go
package main
import "fmt"

type samochod struct {
	marka string
	rok_produkcji int
}

func pokaz(s *samochod) {
	fmt.Printf("marka: %s\n", s.marka)
	fmt.Printf("rok produkcji: %d\n", s.rok_produkcji)
}

func main() {
	var volvo samochod
	
	volvo.marka = "volvo v40"
	volvo.rok_produkcji = 2010

	pokaz(&volvo)
}
